src/server/controllers/VerificationController.ts

```diff
@@ -262,9 +262,12 @@ export default class VerificationController
       addresses: [address],
       contract: checkedContract,
     };
-    const result = await this.verificationService.inject(inputData);
-
-    res.send({ result: [result] });
+    try {
+      const result = await this.verificationService.inject(inputData);
+      res.send({ result: [result] });
+    } catch (e) {
+      throw Error("An error occurred while verifying this contract");
```

Comment:
Why are we catching with a custom message here? This means the error messages inside `Injector` won't be forwarded to the user

Reply:
Is it ok? Because it's a low level message. Do we want to show it?

Reply:
yeah it's right, I'll remove this catch